### ISSUE TITLE
Using `DateTimeOffset` over `DateTime` in `BitChartDemoUtils.cs`

### ISSUE THREAD

**Issue by iPazooki** [bitfoundation/bitplatform#5276]
I reckon, In the [GetNextDays](https://github.com/bitfoundation/bitplatform/blob/be01658c690e16c43987fed23107ad0024ddabe0/src/BlazorUI/Demo/Client/Core/Pages/Components/Chart/BitChartDemoUtils.cs#L38) method, we should use `DateTimeOffset` instead of `DateTime` unless there is a specific reason to use the latter.

- `DateTimeOffset` is typically used for situations where time zone information is important
- `DateTimeOffset` preserves the original local time and offset when serialized or converted to other formats, while `DateTime` may lose or change this information depending on the `DateTimeKind` [property](https://stackoverflow.com/questions/4331189/datetime-vs-datetimeoffset)
- `DateTimeOffset` can be compared with other `DateTimeOffset` values regardless of their offsets, while `DateTime` values can only be compared if they have the same `DateTimeKind` [property](https://stackoverflow.com/questions/4331189/datetime-vs-datetimeoffset)

To summarize, If we need to work with time zones or offsets, we need to use `DateTimeOffset`. If you don’t need this information, or we handle it separately, use `DateTime`.
